CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree iOS SDK Release Notes
 
+## Unreleased
+
+* Luhn validate UnionPay cards
```

Comment:
Should we clarify that Braintree could never process luhn invalid unionpay cards?

Reply:
^ Samesies.

Reply:
review: https://github.com/braintree/braintree_ios/pull/419/commits/5163f234bd7c787b2ab0cb04b8149cd6f72f43c3